src/axolotl/cli/launchers/ray_cluster.py

```
@@ -0,0 +1,339 @@
+            port=port,
+            dashboard_port=dashboard_port,
+            temp_dir=str(temp_dir),
+        )
+
+    state.ssh_user = ssh_user
+    state.ssh_key = ssh_key
+    failures: list[tuple[str, str]] = []
+    joined: list[str] = []
+    for host in workers:
+        LOG.info("joining worker %s", host)
+        result = subprocess.run(  # nosec B603
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)